stubs/app/Actions/Jetstream/InviteTeamMember.php

```diff
@@ -67,8 +67,12 @@ protected function validate($team, string $email, ?string $role)
      */
     protected function rules($team)
     {
+        $class = Jetstream::teamInvitationModel();
+        $teamInvitation = $class::make();
```

Comment:
Where is this `make` method defined?

Reply:
I could be wrong, but I thought that it was on `Illuminate\Database\Eloquent\Model`...although I'm looking now and I can't find it. 😳 Makes me wonder how it's working.

I had originally simply instantiated the model via `new`, but got an error that I couldn't reproduce locally, so I changed it to `Model::make()`....which doesn't appear to be where I thought it was, and yet somehow works.

I'll change it to `new $class`.... 🤓

Reply:
It's odd....locally, with `laravel/framework v8.61.0`, I run the `InviteTeamMemberTest` test, dumping `$teamInvitation` after using `make()` and get the following:
```bash
App\Models\TeamInvitation^ {#3791
  #fillable: array:2 [
    0 => "email"
    1 => "role"
  ]
...
```

Not sure how/why it's working.  After a quick search, I can't find any `make()` method in the API docs or in my own copy of the framework source.  🤔

Anyway...it's changed and still works locally.  Let me know if you have any other questions/concerns. 🤓